frontend/frontend/src/router.js

```diff
@@ -14,6 +15,11 @@ const routes = [
 	{
 		path: "/game",
 		component: Game
+	},
+	{
+		path: "/profile",
```

Comment:
I think when we build in logins and user names this path should dynamically use the users profile name. for now this is good but just a "TODO" note :)

Reply:
Okay, good point. I was thinking we could append it to the path by looking at the global state of a user id but I'm not sure if thats the best way to do it.